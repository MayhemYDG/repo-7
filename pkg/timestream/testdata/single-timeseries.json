{
    "ColumnInfo": [
        {
            "Name": "region",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "cell",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "silo",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "availability_zone",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "microservice_name",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "instance_name",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "process_name",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "jdk_version",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "gc_reclaimed",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": null,
                "TimeSeriesMeasureValueColumnInfo": {
                    "Name": null,
                    "Type": {
                        "ArrayColumnInfo": null,
                        "RowColumnInfo": null,
                        "ScalarType": "DOUBLE",
                        "TimeSeriesMeasureValueColumnInfo": null
                    }
                }
            }
        }
    ],
    "NextToken": null,
    "QueryId": "#QueryId#",
    "Rows": [
        {
            "Data": [
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "ap-northeast-1",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "ap-northeast-1-cell-5",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "ap-northeast-1-cell-5-silo-2",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "ap-northeast-1-3",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "zeus",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "i-zaZswmJk-zeus-0002.amazonaws.com",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "server",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "JDK_11",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": null,
                    "TimeSeriesValue": [
                        {
                            "Time": "2020-08-27 15:09:27.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "68.11868749038354",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:13:41.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "79.27975929378977",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:17:45.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "85.99469252288485",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:21:58.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "25.826528663062486",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:26:09.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "44.034053691200484",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:30:23.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "22.19614040213814",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:34:37.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "65.73055973434877",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:38:54.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "11.636319293712727",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:43:02.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "6.952319429301202",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:47:20.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "51.090158108550284",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:51:40.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "31.364705221330613",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 15:55:57.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "43.93216210516789",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:00:04.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "23.522370881749133",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:04:23.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "47.57745897011495",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:08:31.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "45.28008085322918",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:12:51.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "84.0582541954146",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:17:13.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "77.15655186955061",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:21:34.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "13.7079617339666",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:25:54.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "32.95895451503446",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:30:16.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "45.195109067860365",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:34:59.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "78.77669026675206",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:40:04.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "93.49262995022848",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:45:09.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "59.98831570154902",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:50:13.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "21.834046626056804",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 16:55:13.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "90.59765116867563",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-27 17:00:16.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "43.00633219507514",
                                "TimeSeriesValue": null
                            }
                        }
                    ]
                }
            ]
        }
    ]
}