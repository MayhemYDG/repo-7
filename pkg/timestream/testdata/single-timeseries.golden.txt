ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {}
Name: 
Dimensions: 2 Fields by 26 Rows
+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Name: time                    | Name: gc_reclaimed                                                                                                                                                                                                                                  |
| Labels:                       | Labels: availability_zone=ap-northeast-1-3, cell=ap-northeast-1-cell-5, instance_name=i-zaZswmJk-zeus-0002.amazonaws.com, jdk_version=JDK_11, microservice_name=zeus, process_name=server, region=ap-northeast-1, silo=ap-northeast-1-cell-5-silo-2 |
| Type: []time.Time             | Type: []*float64                                                                                                                                                                                                                                    |
+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| 2020-08-27 15:09:27 +0000 UTC | 68.11868749038354                                                                                                                                                                                                                                   |
| 2020-08-27 15:13:41 +0000 UTC | 79.27975929378977                                                                                                                                                                                                                                   |
| 2020-08-27 15:17:45 +0000 UTC | 85.99469252288485                                                                                                                                                                                                                                   |
| 2020-08-27 15:21:58 +0000 UTC | 25.826528663062486                                                                                                                                                                                                                                  |
| 2020-08-27 15:26:09 +0000 UTC | 44.034053691200484                                                                                                                                                                                                                                  |
| 2020-08-27 15:30:23 +0000 UTC | 22.19614040213814                                                                                                                                                                                                                                   |
| 2020-08-27 15:34:37 +0000 UTC | 65.73055973434877                                                                                                                                                                                                                                   |
| 2020-08-27 15:38:54 +0000 UTC | 11.636319293712727                                                                                                                                                                                                                                  |
| 2020-08-27 15:43:02 +0000 UTC | 6.952319429301202                                                                                                                                                                                                                                   |
| ...                           | ...                                                                                                                                                                                                                                                 |
+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////2AIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAvP3//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADc/f//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAPz9//8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAAAgAAAMABAAAYAAAAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAGwBAABsAQAAAAADAWwBAAACAAAANAAAAAQAAABg/v//CAAAABgAAAAMAAAAZ2NfcmVjbGFpbWVkAAAAAAQAAABuYW1lAAAAAIz+//8IAAAAEAEAAAYBAAB7ImF2YWlsYWJpbGl0eV96b25lIjoiYXAtbm9ydGhlYXN0LTEtMyIsImNlbGwiOiJhcC1ub3J0aGVhc3QtMS1jZWxsLTUiLCJpbnN0YW5jZV9uYW1lIjoiaS16YVpzd21Kay16ZXVzLTAwMDIuYW1hem9uYXdzLmNvbSIsImpka192ZXJzaW9uIjoiSkRLXzExIiwibWljcm9zZXJ2aWNlX25hbWUiOiJ6ZXVzIiwicHJvY2Vzc19uYW1lIjoic2VydmVyIiwicmVnaW9uIjoiYXAtbm9ydGhlYXN0LTEiLCJzaWxvIjoiYXAtbm9ydGhlYXN0LTEtY2VsbC01LXNpbG8tMiJ9AAAGAAAAbGFiZWxzAAAAAAAAgv///wAAAgAMAAAAZ2NfcmVjbGFpbWVkAAASABgAFAAAABMADAAAAAgABAASAAAAFAAAAEQAAABMAAAAAAAACkwAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABAAAAHRpbWUAAAAABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAwAEAAAAdGltZQAAAAAAAAAA/////7gAAAAUAAAAAAAAAAwAFgAUABMADAAEAAwAAACgAQAAAAAAABQAAAAAAAADAwAKABgADAAIAAQACgAAABQAAABYAAAAGgAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAANAAAAAAAAAA0AAAAAAAAAAAAAAAAAAAANAAAAAAAAAA0AAAAAAAAAAAAAAAAgAAABoAAAAAAAAAAAAAAAAAAAAaAAAAAAAAAAAAAAAAAAAAAMbxQIooLxYAMoZkxSgvFgC6DjT+KC8WAFwIHDkpLxYAasyMcykvFgDWYLCuKS8WAEL10+kpLxYADFqqJSovFgC8TWhfKi8WAFBNepsqLxYAeIID2CovFgBC59kTKy8WAChAXE0rLxYAhtqpiSsvFgA2zmfDKy8WAF4D8f8rLxYAGm7xPCwvFgAMPrZ5LC8WADRzP7YsLxYA8N0/8ywvFgA++yM1LS8WAOhlJ3wtLxYAktAqwy0vFgByoPIJLi8WACoFzE8uLxYAQDpYli4vFhBpapOYB1FAEGWGk+fRU0Bv19MKqX9VQFsP6WGX0zlA0AER31sERkCMm+RBNjI2QFuznX3BblBAtp54pMtFJ0C0xhDTLM8bQKrYB02Ki0lAoUtGUl1dP0AgIn4WUfdFQMZ+HRm6hTdAeLbvLOrJR0B0bXyw2aNGQEYKzm+6A1VAPvYh8gRKU0B5adz1eWorQFg2hAW/ekBAgc58VfmYRkCVtBdLtbFTQLtQxT+HX1dAXyMAIYH+TUDMQGYUhNU1QDH4r+o/plZAO2FNfs+ARUAQAAAADAAUABIADAAIAAQADAAAABAAAAAsAAAAOAAAAAAAAwABAAAA6AIAAAAAAADAAAAAAAAAAKABAAAAAAAAAAAAAAAAAAAAAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAvP3//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADc/f//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAPz9//8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAAAgAAAMABAAAYAAAAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAGwBAABsAQAAAAADAWwBAAACAAAANAAAAAQAAABg/v//CAAAABgAAAAMAAAAZ2NfcmVjbGFpbWVkAAAAAAQAAABuYW1lAAAAAIz+//8IAAAAEAEAAAYBAAB7ImF2YWlsYWJpbGl0eV96b25lIjoiYXAtbm9ydGhlYXN0LTEtMyIsImNlbGwiOiJhcC1ub3J0aGVhc3QtMS1jZWxsLTUiLCJpbnN0YW5jZV9uYW1lIjoiaS16YVpzd21Kay16ZXVzLTAwMDIuYW1hem9uYXdzLmNvbSIsImpka192ZXJzaW9uIjoiSkRLXzExIiwibWljcm9zZXJ2aWNlX25hbWUiOiJ6ZXVzIiwicHJvY2Vzc19uYW1lIjoic2VydmVyIiwicmVnaW9uIjoiYXAtbm9ydGhlYXN0LTEiLCJzaWxvIjoiYXAtbm9ydGhlYXN0LTEtY2VsbC01LXNpbG8tMiJ9AAAGAAAAbGFiZWxzAAAAAAAAgv///wAAAgAMAAAAZ2NfcmVjbGFpbWVkAAASABgAFAAAABMADAAAAAgABAASAAAAFAAAAEQAAABMAAAAAAAACkwAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABAAAAHRpbWUAAAAABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAwAEAAAAdGltZQAAAAAAAwAAQVJST1cx
